### DISCUSSION TITLE
zkSync plugin is not work in remix

### DISCUSSION THREAD

**Discussion by zhouhangmyers** [Cyfrin/foundry-full-course-cu#1965] [Q&A]
i have already created the contracts folder and put the file FundMe.sol into its, and i hit the compile but the Depoly is still not work.
![2f988968d8c0d36d64970115fd2a9c81](https://github.com/Cyfrin/foundry-full-course-cu/assets/151934937/00f58efe-495f-4c34-985f-804e180062b2)

**Comment by EngrPips** [comment]
Please refresh your browser, then retry.

**Comment by ciaranightingale** [comment]
Hi @zhouhangmyers did @EngrPips suggestion work for you?

**Comment by alperenc** [comment]
I have the same issue and refresh did not work.

**Reply by EngrPips** [reply]
The plug-in sometimes don't work properly so you should try again at a later time. Don't let this slow you down as Patrick mention many times that zksync stuff is optional.

**Comment by Haardtripathi** [comment]
Hello @zhouhangmyers , please first show the "compile" box of the zksync?

**Comment by EngrPips** [comment]
@alperenc  if you haven't resolved this issue please provide more details on the issue you are running so that @Haardtripathi  can help.

**Comment by alperenc** [comment]
<img width="1358" alt="Screenshot 2024-08-30 at 5 33 01 PM" src="https://github.com/user-attachments/assets/71499236-fb14-4cd2-b037-d63e47102293">
<img width="744" alt="Screenshot 2024-08-30 at 5 34 19 PM" src="https://github.com/user-attachments/assets/9c7c77c5-f154-44a1-a310-b48d9a7644a2">

No, not resolved. Still not available for deployment in Zksync plugin after compilation is successful. Attached some screenshots.

**Reply by EngrPips** [reply]
Hey @alperenc, Please switch your workspace to `default_workspace` and ensure all your files are inside the `contracts` folder, then retry.

**Reply by alperenc** [reply]
Did not work.

<img width="734" alt="Screenshot 2024-08-30 at 6 08 45 PM" src="https://github.com/user-attachments/assets/d12b2253-57c9-4a39-a0f1-612547c4a172">
<img width="735" alt="Screenshot 2024-08-30 at 6 09 07 PM" src="https://github.com/user-attachments/assets/c2f909ed-6190-4085-96dd-8470289701ee">

**Reply by EngrPips** [reply]
What chain are you currently working on?

**Reply by alperenc** [reply]
I don't think I understand the question. I'm trying to deploy on the zkSync Sepolia?

**Reply by EngrPips** [reply]
It is supposed to work. I am unsure why it is not working. Maybe refresh your browser and retry if it still doesn't work. You can try it with another browser, and if the same issue persists, I will say not to let that slow you down, as `zksync` stuff is optional.

**Comment by AboloreDev** [comment]
Try refreshing your browser and retry. If this doesnt work, create a new folder for contract folder and add the fundMe.sol into it. then try redeployment on zksync plugIn once again

**Comment by Haardtripathi** [comment]
@zhouhangmyers 
<img width="356" alt="image" src="https://github.com/user-attachments/assets/56655233-1a80-4284-8c19-749aeaee03be">
Does this show up after compilation?
